README.md

```diff
@@ -104,6 +104,7 @@ maintain them and address inconsistencies:
 | `doom-laserwave`                    | a clean 80's synthwave / outrun theme inspired by VS Code's [laserwave][laserwave] (thanks to [hyakt])       |
 | `doom-manegarm`                     | an original autumn-inspired dark theme (thanks to [kenranunderscore])                                        |
 | `doom-material`                     | adapted from [Material Themes] (thanks to [tam5])                                                            |
+| `doom-meltbus`                      | a dark, mostly monochromatic theme (thanks to [spacefrogg])                                                  |
```

Comment:
Also, It seems that you haven't added a markdown link to your github page under `[spacefrogg]` (see the bottom of the raw `README.md` file). Can you do that as well?

Reply:
Sure!